### PR TITLE
Fix example page title and description

### PR DESCRIPTION
**Pull request by Anton-4**
Fixes example description "Example of index.html in the Roc programming language." etc. discussed in #6741

### CODE REVIEW

File: www/build.sh [hunk 1]
```diff
@@ -18,11 +18,12 @@ cd $SCRIPT_RELATIVE_DIR
 rm -rf build/
 cp -r public/ build/
 
+rm -rf content/examples/
 # download the latest code for the examples
 echo 'Downloading latest examples...'
 curl -fL -o examples-main.zip https://github.com/roc-lang/examples/archive/refs/heads/main.zip
 rm -rf examples-main/
-unzip -o examples-main.zip
+unzip -o -q examples-main.zip
```
Comment (Anton-4) [inline www/build.sh:26]: q for less verbose output

File: www/build.sh [hunk 2]
```diff
@@ -97,7 +98,7 @@ fi
 $roc version
 
 echo 'Building site markdown content'
-$roc run www/main.roc -- www/content/ www/build/
+$roc dev www/main.roc -- www/content/ www/build/
```
Comment (Anton-4) [inline www/build.sh:101]: We don't want to run this regardless of errors!

File: www/main.roc [hunk 3]
```diff
@@ -29,10 +30,12 @@ getPageInfo = \pagePathStr ->
...
-                Str.splitLast pagePathStr "/"
-                |> unwrapOrCrash "This splitLast should never fail. pagePathStr ($(pagePathStr)) did not contain any `/`."
-                |> .after # get part after last /
-                |> (\pageTitle -> { title: pageTitle, description: "Example of $(pageTitle) in the Roc programming language." })
+                Str.split pagePathStr "/"
+                |> List.takeLast 2
+                |> List.first # we use the folder for name for the page title, e.g. Json from examples/Json/README.html
+                |> unwrapOrCrash "This List.first should never fail. pagePathStr ($(pagePathStr)) did not contain any `/`."
+                |> (\pageTitle ->
+                    { title: "$(pageTitle) | Roc", description: "$(pageTitle) example in the Roc programming language." })
```
Comment (Anton-4) [inline www/main.roc:38]: Fixed this to use the folder name instead of README.html.

File: www/main.roc [hunk 4]
```diff
@@ -45,17 +48,9 @@ unwrapOrCrash = \result, errorMsg ->
         Err err ->
             crash "$(Inspect.toStr err): $(errorMsg)"
 
-getTitle : Str -> Str
-getTitle = \current ->
-    getPageInfo current |> .title
-
-getDescription : Str -> Str
-getDescription = \current ->
-    getPageInfo current |> .description
```
Comment (Anton-4) [inline www/main.roc:54]: These end up calling `getPageInfo` twice, which is unnecessary.

**Review by ayazhafiz — APPROVED** [review]

**Pull request merged**